packages/cosmic-swingset/lib/ag-solo/start.js

```diff
@@ -297,6 +297,26 @@ export default async function start(basedir, argv) {
     resetOutdatedState,
   } = d;
 
+  // Remove wallet traces.
+  await unlink('html/wallet').catch(_ => {});
+
+  const { wallet } = JSON.parse(fs.readFileSync('options.json', 'utf-8'));
```

Comment:
Is there sense in making this robust to the working directory, or are you comfortable with that being implied by the `yarn start` workflow driving?

Reply:
> Is there sense in making this robust to the working directory, or are you comfortable with that being implied by the `yarn start` workflow driving?

This command can only run after an `agoric init <dirname>; cd <dirname>` and it heuristically identifies when you're definitely not in such a directory.

But thanks for asking.